dev/kyuubi-codecov/pom.xml

```diff
@@ -95,43 +95,31 @@
 
         <dependency>
             <groupId>org.apache.kyuubi</groupId>
-            <artifactId>kyuubi-flink-sql-engine_${scala.binary.version}</artifactId>
```

Comment:
as a trade-off, codecov does not cover engines other than spark

Reply:
it is a big drawback for code quality, and I am hesitating

Reply:
Is there any reason we must remove these dependencies?

Reply:
Because we propose running different engines in different CI jobs, then it does not make sense to include the modules which have ut but not run